tutorials/katacoda/thanos/2-lts/index.json

```diff
@@ -1,17 +1,54 @@
 {
   "title": "Intro: Downsampling and unlimited metric retention for Prometheus",
   "description": "Learn how to extend you metric retention in a cheap way with Thanos.",
+  "difficulty": "Beginner",
+  "time": "15-20 Minutes",
   "details": {
     "steps": [
+      {
+        "title": "Start initial Prometheus servers",
+        "text": "step1.md",
+        "verify": "step1-verify.sh"
+      },
+      {
+        "title": "Thanos Sidecars",
+        "text": "step2.md",
+        "verify": "step2-verify.sh"
+      },
+      {
+        "title": "Configuring Object store",
+        "text": "step3.md",
+        "verify": "step3-verify.sh"
+      },
+      {
+        "title": "Thanos Store Gateway",
+        "text": "step4.md",
+        "verify": "step4-verify.sh"
+      },
```

Comment:
Then here we can another querier or configure querier (better) to use this store as well